Client/actions/eventActions.js

```diff
@@ -0,0 +1,7 @@
+import axios from 'axios';
+
+export function createEvent(event) {
```

Comment:
action creators like this one need to dispatch some actions. This one calls backend but does not do anything with the result, maybe it would be better like this
```
axios.post('/api/events', event)
  .then((response) => dispatch({ type: 'EVENT_CREATED', payload: response });

or something like that...